apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx

```diff
@@ -39,10 +39,7 @@ export const UsePoolerCheckbox = ({
 
   const { data, isLoading, isSuccess } = usePoolingConfigurationQuery({ projectRef })
   const { data: settings, isSuccess: isSuccessSettings } = useProjectSettingsQuery({ projectRef })
-  const { data: pgBouncerStatus } = usePgBouncerStatus({
-    projectRef: projectRef!,
-    tld: settings ? settings.project.db_host.split('.').pop() : 'co',
-  })
+  const { data: pgBouncerStatus } = usePgBouncerStatus({ projectRef: projectRef })
```

Comment:
Should this query be disabled on self-hosted?

Reply:
great point! yes it should